forch/faucetizer.py

```diff
@@ -131,31 +135,38 @@ def _faucetize(self):
             for port, port_map in switch_map.get('interfaces', {}).items():
                 if 'stack' in port_map or 'lacp' in port_map or 'output_only' in port_map:
                     continue
-                port_map['native_vlan'] = self._config.unauthenticated_vlan
-
-        devices = {**self._dynamic_devices, **self._static_devices}
-        for mac, device in devices.items():
-            if device.placement.switch and device.behavior.segment:
-                switch_cfg = behavioral_faucet_config.get('dps', {}).get(
-                    device.placement.switch, {})
-                port_cfg = switch_cfg.get('interfaces', {}).get(device.placement.port)
-
-                if not port_cfg:
-                    LOGGER.warning('Switch or port not defined in faucet config: %s, %s',
-                                   device.placement.switch, device.placement.port)
-                    continue
 
-                vid = self._segments_to_vlans.get(device.behavior.segment)
-                if not vid:
-                    LOGGER.warning('Device segment does not have a matching vlan: %s, %s',
-                                   mac, device.behavior.segment)
-                    continue
-
-                port_cfg['native_vlan'] = vid
-                if device.behavior.role:
-                    port_cfg['acls_in'] = [f'role_{device.behavior.role}', 'tail_acl']
-                else:
-                    port_cfg['acls_in'] = ['tail_acl']
+                if self._config.unauthenticated_vlan:
```

Comment:
We need to make sure we're starting to add proper unit testing to changes like this -- esp. when they have algorithmic behavior behind them (which this does).  Until we get the unit test framework in place, can you write some simple test functions that just check some of the cases (e.g. both static & dynamic placements work together properly) and then call them at the end in the main section?

Reply:
Reverted this change. I think it is better to make this behavior change in another PR...